Makefile

```diff
@@ -107,9 +107,11 @@ node-version: node_modules/semver
 
 # ensures that the `node_modules` directory is installed and up-to-date with
 # the dependencies listed in the "package.json" file.
-node_modules: package.json | node-version
+node_modules/*: package.json
```

Comment:
Yep, just hit a really weird case. Gonna have to rethink this. I would explain more, but currently 2:30am and have to sleep.